src/test/java/org/savantbuild/dep/DefaultDependencyServiceTest.java

```diff
@@ -293,47 +293,56 @@ public void buildGraphWithExclusions() {
   }
 
   @Test
-  public void buildGraphWithTransitiveAndDirectExclusions() {
```

Comment:
Did we lose a test here?

Reply:
No, just the project's IntellIj sorting/reordering rules moved it to line 385.